SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -7,26 +7,34 @@
 //
 
 import UIKit
-import DZNEmptyDataSet
 
 class ViewController: UIViewController {
     
-    private let feedbackTableView = UITableView(frame: .zero)
-    private let searchController = UISearchController() // TODO: For later
+    private var headerCollectionView: UICollectionView!
+    private var feedbackCollectionView: UICollectionView!
+    private let searchController = UISearchController(searchResultsController: nil) // TODO: For later
         
+    private let headersData = ["Customer Service", "Bugs & Requests"]
     private var feedbackData = [Feedback]()
-    private var filteredFeedbackData = [Feedback]() // TODO: For later
+    private var filteredFeedbackData = [Feedback]()
     
     private(set) var countEditTaps: Int = 0
+    
+    private var isTwoway: Bool {
```

Comment:
Can we make this a function that j takes a Feedback object and returns a boolean instead? Confused about this indexing

Reply:
Maybe even have that as an instance method of the feedback class so we can just do `feedback.isTwoWay()`

Reply:
This variable doesn't say whether or not the feedback itself `isTwoWay` but rather whether the current header is on Customer Service and thus only TwoWay feedback should be shown.